src/pages/Home.jsx

```diff
@@ -1,6 +1,136 @@
+                <div className="relative z-10 flex flex-col items-center justify-center h-full text-black">
+                    <nav className="absolute top-0 left-0 w-full flex justify-around items-center py-4">
+                        <a href="#home" className="text-black">HOME</a>
+                        <a href="#about" className="text-black">ABOUT</a>
+                        <a href="#projects" className="text-black">PROJECTS</a>
+                        <img src={FusionLogo} alt="Logo" className="h-8" />
+                        <a href="#board" className="text-black">BOARD</a>
+                        <a href="#programs" className="text-black">PROGRAMS</a>
+                        <a href="#meetings" className="text-black">MEETINGS</a>
+                    </nav>
+                    <h1 className="text-6xl font-bold">FUSION</h1>
+                    <h2 className="text-2xl mt-4">FILIPINX UNDERGRADUATE SCIENTISTS-ENGINEERS IN AN ORGANIZED NETWORK</h2>
+                </div>
+            </div>
+            {/*What is Fusion*/}
+            <div className="bg-white py-20">
+                <h2 className="text-4xl font-semibold text-center text-black">WHAT IS FUSION?</h2>
+                <p className="mt-8 mx-auto max-w-3xl text-center text-black">
+                    Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.
+                </p>
+            </div>
+            {/*Image Carousel*/}
+            <div className="carousel w-full flex justify-center space-x-4">
+                {/* Slide 1 */}
+                <div id="slide1" className="carousel-item relative w-1/3">
+                    <img src={placeholderDarkImage} className="w-full h-64 object-cover" alt="Slide 1" />
+                </div>
+                {/* Slide 2 */}
+                <div id="slide2" className="carousel-item relative w-1/3">
+                    <img src={placeholderDarkImage} className="w-full h-64 object-cover" alt="Slide 2" />
+                </div>
+                {/* Slide 3 */}
+                <div id="slide3" className="carousel-item relative w-1/3">
+                    <img src={placeholderDarkImage} className="w-full h-64 object-cover" alt="Slide 3" />
+                </div>
+                {/* Slide 4 */}
+                <div id="slide4" className="carousel-item relative w-1/3">
+                    <img src={placeholderDarkImage} className="w-full h-64 object-cover" alt="Slide 4" />
+                </div>
+            </div>
+            {/*What does Fusion do?*/}
+            <div className="fusion-container py-20">
```

Comment:
There are some differences between the color, might need to fix this within the css module for this file, text color also needs to be specified to be black

![image](https://github.com/benson-fm/FUSION-Website/assets/72283466/43245b84-999e-491f-a3e4-e105cca9df72)